pengwin-create-targz.sh

```diff
@@ -31,10 +35,10 @@ sudo chroot $DIST /bin/bash -c "echo 'Defaults lecture_file = /etc/sudoers.lectu
 sudo chroot $DIST /bin/bash -c "echo 'Enter your UNIX password below. This is not your Windows password.' > /etc/sudoers.lecture"
 
 # remove unnecessary packages in initial image
-sudo chroot $DIST apt-get -y -q remove systemd dmidecode --allow-remove-essential
+#sudo chroot $DIST apt-get -y -q remove systemd dmidecode --allow-remove-essential
 
 # clean up any orphaned apt dependencies
-sudo chroot $DIST apt-get -y -q autoremove
+#sudo chroot $DIST apt-get -y -q autoremove
```

Comment:
This helps to have a smaller image, why is being commented?

Reply:
It was trying to remove required packages/dependencies that was causing apt to throw errors. There's probably a better resolution than commenting out autoremove, but I'm going to work on getting the build process to work with the non-legacy rootfs repo you mentioned before working on this PR further. If I can't get it to work, I'll fall back on this one (which uses cdebootstrap instead of debootstrap)